### PR TITLE
Api deprecation changes

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[Api deprecation changes]

### Issues Resolved
[]

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: ov/ov.go [hunk 1]
```diff
@@ -44,6 +46,33 @@ func (c *OVClient) NewOVClient(user string, password string, domain string, endp
 			IfMatch:    ifmatch,
 		},
 	}
+	apiver, err := c.GetAPIVersion()
+	//If no api version is provided use the current version to create client
+	if apiversion == 0 {
```
Comment (chebroluharika) [inline ov/ov.go:51]: I am not sure about the case where we get apiversion as 0

File: ov/ov.go [hunk 2]
```diff
@@ -20,19 +20,21 @@ package ov
 import (
 	"errors"
 	"fmt"
-
 	"github.com/HewlettPackard/oneview-golang/rest"
 	"github.com/docker/machine/libmachine/log"
 )
 
+var min_api_version int = 600
```
Comment (chebroluharika) [inline ov/ov.go:27]: Please remove this line as it doesn't have any usage

**Review by chebroluharika — APPROVED** [review]

File: ov/ov.go (hunk 1 above)
Comment (nabhajit-ray) [reply ov/ov.go:51]: If no apiverison is given in config file 0 stored in the config struct

**Review by VenkateshRavula — APPROVED** [review]

File: ov/ov.go (hunk 2 above)
Comment (nabhajit-ray) [reply ov/ov.go]: Done

**Pull request merged**